app/controllers/current_user_controller.rb

```diff
@@ -0,0 +1,6 @@
+class CurrentUserController < ApplicationController
+  before_action :authenticate_user!
+  def index
+    render json: current_user, status: :ok
```

Comment:
Hi guys. Im curious as to how exactly we're using this and how necessary this method is.

Our API should be stateless, meaning it shouldn't be the one keeping state of a user's authenticated condition. Therefore, having a "current_user" in the backend might not seem intuitive.

Reply:
:heavy_check_mark: Removed! 
It was not needed, indeed. :+1:

Reply:
Great!!